tritium-jmh/build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'me.champeau.gradle:jmh-gradle-plugin:0.5.2'
+        classpath 'me.champeau.gradle:jmh-gradle-plugin:0.5.3'
```

Comment:
https://github.com/melix/jmh-gradle-plugin/blob/12ae9578ddad3c038cfeab9fdbbc37b784715f05/CHANGELOG.txt#L1-L6

Version 0.5.3
-------------
    * Support for Gradle 7
    * Default duplicates strategy is now INCLUDE instead of FAIL, for backwards compatibility
    * Rename `jmhRuntime` to `jmhRuntimeClaspath` for clarity
    * Upgrade to JMH 1.27